### PR TITLE
Print name of path that is deemed too long.

### PR DESCRIPTION
**Pull request by akesandgren**
Makes it easier to see where the problem is...

### CODE REVIEW

**Review by daltonbohning — COMMENTED** [review]
Within this file, there are three total instances of the error `Path name is too long`.
Maybe we should update the other 2 instances to use this new format as well?

**Comment by akesandgren** [comment]
Probably, I picked the one that triggered on my run and didn't see any others at the moment.

File: src/common/mfu_flist_walk.c [hunk 1]
```diff
@@ -488,7 +488,7 @@ static void walk_stat_process_dir(char* dir, CIRCLE_handle* handle)
                 }
                 else {
                     /* name is too long */
-                    MFU_LOG(MFU_LOG_ERR, "Path name is too long: %lu chars exceeds limit %lu", len, sizeof(newpath));
+                    MFU_LOG(MFU_LOG_ERR, "Path name is too long: %lu chars exceeds limit %lu, (%s/%s)", len, sizeof(newpath), dir, name);
```
Comment (adammoody) [inline src/common/mfu_flist_walk.c:491]: This is good, but let's tweak the format just a bit.  I try to use a pseudo-convention to always contain printed path names in single quotes set off by a colon.
```suggestion
                    MFU_LOG(MFU_LOG_ERR, "Path name is too long, %lu chars exceeds limit %lu: '%s/%s'", len, sizeof(newpath), dir, name);
```

**Comment by adammoody** [comment]
@akesandgren , would you mind also picking up the other instances in this one commit?

File: src/common/mfu_flist_walk.c (hunk 1 above)
Comment (akesandgren) [reply src/common/mfu_flist_walk.c]: @adammoody  done.

**Comment by adammoody** [comment]
Thanks @akesandgren

**Pull request merged**